docs/en/UI/Angular/Theme-Configurations.md

```diff
@@ -1,3 +1,220 @@
 # Angular UI: Theme Configurations
 
-> This document is in draft version.
\ No newline at end of file
+Theme packages no longer import styles as CSS modules as of ABP version 6.0. The following style setting needs to be included to the angular.json file in order for theme packages to load styles.
+## Lepton X Lite
+```json
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/bootstrap-dim.css",
+    "inject": false,
+    "bundleName": "bootstrap-dim"
+},
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/ng-bundle.css",
+    "inject": false,
+    "bundleName": "ng-bundle"
+},
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/side-menu/layout-bundle.css",
+    "inject": false,
+    "bundleName": "layout-bundle"
+},
+{
+    "input": "node_modules/@abp/ng.theme.lepton-x/assets/css/abp-bundle.css",
+    "inject": false,
+    "bundleName": "abp-bundle"
+},
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/bootstrap-dim.rtl.css",
+    "inject": false,
+    "bundleName": "bootstrap-dim.rtl"
+},
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/ng-bundle.rtl.css",
+    "inject": false,
+    "bundleName": "ng-bundle.rtl"
+},
+{
+    "input": "node_modules/@volo/ngx-lepton-x.lite/assets/css/side-menu/layout-bundle.rtl.css",
+    "inject": false,
+    "bundleName": "layout-bundle.rtl"
+},
+{
+    "input": "node_modules/@abp/ng.theme.lepton-x/assets/css/abp-bundle.rtl.css",
+    "inject": false,
+    "bundleName": "abp-bundle.rtl"
+},
+```
+
+## Theme Basic
+```json
+{
+    "input": "node_modules/bootstrap/dist/css/bootstrap.rtl.min.css",
+    "inject": false,
+    "bundleName": "bootstrap-rtl.min"
+},
+{
+    "input": "node_modules/bootstrap/dist/css/bootstrap.min.css",
+    "inject": true,
+    "bundleName": "bootstrap-ltr.min"
```

Comment:
```suggestion
    "bundleName": "bootstrap.min"
```

Reply:
No, the theme-basic searches for a direction prefix in the bundle name to determine which bundle will load. The [source code](https://github.com/abpframework/abp/blob/ea6cdfd8701421d9af221d2d2480e876d3db64bd/npm/ng-packs/packages/theme-basic/src/lib/handlers/lazy-style.handler.ts#L5)